Extensions/Lighting/lightruntimeobject.js

```diff
@@ -201,8 +186,8 @@ gdjs.LightRuntimeObject.prototype.getDebugMode = function () {
 
 /**
  * Returns PIXI.Texture if it exists, null otherwise.
- * @returns {?PIXI.Texture} the texture, if any, null otherwise.
+ * @returns {string} the path of texture.
```

Comment:
This will be ?string too :)

Reply:
Actually no sorry, this string is always defined, it's just empty when not defined.